src/highlight/HighlightListener.ts

```diff
@@ -54,10 +53,8 @@ export default class HighlightListener {
         // Clear previous selection
         this.store.dispatch(setSelectionAction(null));
 
-        if (!getIsInitialized(this.store.getState())) {
-            this.annotatedEl.addEventListener('mousedown', this.handleMouseDown);
-            this.annotatedEl.addEventListener('mouseup', this.handleMouseUp);
-        }
+        this.annotatedEl.addEventListener('mousedown', this.handleMouseDown);
```

Comment:
This isn't really the "annotated element" anymore. Maybe "parentEl" or something similar?

Reply:
Or maybe just `textLayerEl`? Or `element`